.github/renovate.json5

```
@@ -130,5 +130,15 @@
       ],
       "enabled": false
     }
-  ]
+  ],
+  postUpgradeTasks: {
+// Post-upgrade tasks that are executed before a commit is made by Renovate.
+    "commands": [
+      "make generate"
+    ],
+    fileFilters: [
+      ["**/*"]
+    ],
```

I [think](https://docs.renovatebot.com/configuration-options/#filefilters) it's only an array of strings, not an array of arrays:
```
 INFO: Validating .github/renovate.json5
ERROR: Found errors in configuration
       "file": ".github/renovate.json5",
       "errors": [
         {
           "topic": "Configuration Error",
           "message": "Invalid configuration option: postUpgradeTasks.fileFilters[0].0"
         }
       ]
```
```suggestion
    fileFilters: ["**/*"],
```

---

Oh really sorry I must've left it unintentionally, yes it should be an array indeed.